main.go

```diff
@@ -44,11 +46,9 @@ var (
 	mouse            bool
 
 	rootCmd = &cobra.Command{
-		Use:   "glow [SOURCE|DIR]",
-		Short: "Render markdown on the CLI, with pizzazz!",
-		Long: paragraph(
-			fmt.Sprintf("\nRender markdown on the CLI, %s!", keyword("with pizzazz")),
-		),
+		Use:              "glow [SOURCE|DIR]",
+		Short:            "Render markdown on the CLI, with pizzazz!",
+		Long:             fmt.Sprintf("Render markdown on the CLI, %s!", keyword("with pizzazz")),
 		SilenceErrors:    false,
 		SilenceUsage:     true,
```

Comment:
Couldn't fang.Execute sets this two?

BTW, do you really want SilentErrors to be false?

Reply:
it does, should remove it